core/src/test/java/org/bitcoinj/core/TransactionTest.java

```diff
@@ -278,6 +278,33 @@ public void testCLTVPaymentChannelTransactionRefund() {
     }
 
     @Test
+    public void canParseTransaction() {
```

Comment:
Maybe rename this test to something more meaningful, e.g. `witnessTransaction()`?